pkg/crc/cluster/cluster.go

```diff
@@ -77,3 +92,62 @@ func GetRootPartitionUsage(sshRunner *ssh.SSHRunner) (int64, int64, error) {
 	}
 	return diskSize, diskUsage, nil
 }
+
+func AddPullSecret(sshRunner *ssh.SSHRunner, oc oc.OcConfig, pullSec string) error {
+	if err := addPullSecretToInstanceDisk(sshRunner, pullSec); err != nil {
+		return err
+	}
+
+	base64OfPullSec := base64.StdEncoding.EncodeToString([]byte(pullSec))
+	cmdArgs := []string{"patch", "secret", "pull-secret", "-p",
+		fmt.Sprintf(`{"data":{".dockerconfigjson":"%s"}}`, base64OfPullSec),
+		"-n", "openshift-config", "--type", "merge"}
+	logging.Debugf("Executing oc %s\n", strings.Join(cmdArgs, " "))
```

Comment:
This belongs to oc.RunOcCommand rather than here if you want such a log.
`logging.Debugf("Patching cluster config for new pull secret")` for a debug log which goes in this method.
This is also leaking the pull secret to debug logs.